app/views/maintenance_tasks/tasks/show.html.erb

```diff
@@ -16,6 +16,16 @@
           <%= form.file_field :csv_file %>
         </div>
       <% end %>
+      <% if @task.params.any? %>
+        <div class="block">
+          <%= form.fields_for :task_params do |ff| %>
+            <% @task.params.each do |param| %>
+              <%= ff.label "#{param}: ", class: "label" %>
```

Comment:
I'm not sure if the label would tie to the text field correctly as it stands? We may need to do this:

```suggestion
              <%= ff.label param, "#{param}:", class: "label" %>
```

Reply:
Actually, even in omitting the name, it seems to match the label to the text field 😄 
![Screen Shot 2021-05-14 at 10 25 27 AM](https://user-images.githubusercontent.com/22918438/118284719-b639b600-b49e-11eb-8f4f-56ea07216708.png)
I guess that's typical ✨ Rails Magic ✨ for ya.